StarkInfra/StarkInfra/CreditPreview/CreditPreview.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    /// <summary>
+    /// CreditPreview object
+    /// <br/>
+    /// A CreditPreview is used to get information from a credit before taking it.
+    /// This resource can be used to preview credit notes
+    /// <br/>
+    /// Properties:
+    /// <list>
+    ///     <item>Credit [CreditNotePreview]: Information preview of the informed credit.</item>
+    ///     <item>Type [string]: Credit type. ex: "credit-note"</item>
```

Comment:
Docstring do parâmetro **type** seria `Credit type, inferred from the payment parameter if it is not a dictionary. ex: "credit-note"`